docs/INSTALL.md

```diff
@@ -236,15 +236,15 @@ Run all unit tests:
 
 ```bash
 cd build
-../tools/python/run_desktop_tests.py -f . -u ../data/ -d ../data/
+ctest -LE "fixture" --output-on-failure
```

Comment:
What output will be seen on failure without `--output-on-failure` key? It is inconvenient to always specify this option.

Reply:
Nothing. Ctest will store results under the `build/Testing/Temporary/LastTest.log`
It is also inconvenient to run this command from terminal.
IDEs provide great functionality to run tests.
Example from CLion:
<img width="1150" alt="image" src="https://github.com/organicmaps/organicmaps/assets/10351358/44327ee4-d3f9-4827-9b62-b3f80a4666a1">

And you can configure it in the way you like:
<img width="798" alt="image" src="https://github.com/organicmaps/organicmaps/assets/10351358/0fdd869b-9597-4a8c-9d60-e04acee75762">